example/lib/main.dart

```diff
@@ -32,8 +32,6 @@ bool get isDesktop {
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  SystemTheme.accentInstance;
```

Comment:
I'm not sure about that, this had an utility ???

Reply:
Yes! It's used to get the current accent color of the device.